src/test/java/core/basesyntax/StrategyOperationsTest.java

```diff
@@ -0,0 +1,152 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.NotEnoughFruitsException;
+import core.basesyntax.exceptions.NullDataException;
+import core.basesyntax.models.FruitTransaction;
+import core.basesyntax.service.OperationHandler;
+import core.basesyntax.service.strategy.operations.BalanceStrategy;
+import core.basesyntax.service.strategy.operations.PurchaseStrategy;
+import core.basesyntax.service.strategy.operations.ReturnStrategy;
+import core.basesyntax.service.strategy.operations.SupplyStrategy;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class StrategyOperationsTest {
```

Comment:
I suggest to create separate test classes for each OperationHandler. If logic in handlers will be expanded and more tests will be required it will be hard to find the necessary test here. Or you can create a nested test class